tasks/esbuild-packages.js

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env node
+
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import fg from 'fast-glob';
+import { build } from 'esbuild';
+import fs from 'fs';
+
+const makeDev = {
+    name: 'make-dev',
+    setup(build) {
+        // Load ".txt" files and return an array of words
+        build.onLoad({ filter: /\.ts$/ }, async (args) => {
+            let js = await fs.promises.readFile(args.path, 'utf8');
+            const relativeImportRegex = RegExp(
+                'import(.+)["|\'](?![a-zA-Z@])(..+).js["|\'];',
+                'g'
+            );
+            const contents = js.replace(
+                relativeImportRegex,
+                "import$1'$2.dev.js'"
+            );
+            return {
+                contents,
+                loader: 'ts',
+            };
+        });
+    },
+};
+
+const buildPackage = (paths, options) => {
+    const devPaths = paths.filter(
+        (path) =>
+            path.search('/test/') === -1 && path.search('/stories/') === -1
+    );
+    build({
+        entryPoints: devPaths,
+        bundle: false,
+        outdir: '.',
+        sourcemap: true,
+        watch: options.watch,
+        define: { 'window.__SWC_DEBUG__': true },
```

Comment:
What would you think of using __DEBUG__ instead so that Hera could use the same naming approach?

Reply:
I could definitely be convinced. There's some level of copy pasta not making them the same _could_ prevent. This could be made an argument of the method for easy cross project usage, as well. My first goal was to make sure I could get the build not to blow up at types time. Next goal is to see if I can keep that happening while being able to remove `window.` as it bothers my and makes the placement code pretty verbose. If we can get that working, I'd love to get back to just `DEBUG` as it's simpler and less likely to cause issues, and solves your request, too!